newrelic/structures_newrelic_one_dashboard.go

```diff
@@ -571,6 +576,38 @@ func expandDashboardTableWidgetConfigInitialSortingInput(w map[string]interface{
 	return nil
 }
 
+func expandDashboardTableWidgetConfigDataFormatInput(w map[string]interface{}) []*dashboards.DashboardWidgetDataFormat {
+	var tableWidgetDataFormat []*dashboards.DashboardWidgetDataFormat
+	if q, ok := w["data_format"]; ok && len(q.([]interface{})) >= 1 && q.([]interface{})[0] != nil {
```

Comment:
```
&& len(q.([]interface{})) >= 1 && q.([]interface{})[0] != nil
```

now that `MinItems:1` is set in the schema, we might not need this. Please give this a try and see if we can discard this condition.

Reply:
yeah ..the condition is no longer needed as the schema of data_format demands for mandatory fields if mentioned which ensures atleast one list item is present